api/daemon_test.go

```diff
@@ -51,13 +51,15 @@ func TestSignedUpdate(t *testing.T) {
 
 	// newer version
 	uh.version = "100.0"
-	st.getAPI("/daemon/updates/check", &info)
-	if !info.Available {
+	err = st.getAPI("/daemon/updates/check", &info)
+	if err != nil {
+		t.Error(err)
+	} else if !info.Available {
 		t.Error("new version should be available")
 	}
 
 	// apply (bad signature)
```

Comment:
means the apply call is being tested with a bad signature, which should cause a failure